addons/social_share/models/res_country.py

```diff
@@ -0,0 +1,7 @@
+from odoo import models
+
+class Country(models.Model):
+    _inherit = 'res.country'
+
+    def _snshare_allowed_fields(self):
+        return ['name']
```

Comment:
Could also include display_name each time we include name, just to let people use the "name_get" version of the record